ERROR - 2025-02-26 05:38:41 --> mysqli_sql_exception: Table 'edocs.auth' doesn't exist in engine in C:\xampp\htdocs\CI_CRUD_MVC\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CI_CRUD_MVC\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CI_CRUD_MVC\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CI_CRUD_MVC\app\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CI_CRUD_MVC\app\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
#8 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(933): App\Controllers\UsersController->users()
#9 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#10 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI_CRUD_MVC\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI_CRUD_MVC\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\CI_CRUD_MVC\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\CI_CRUD_MVC\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-02-26 05:38:41 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'edocs.auth' doesn't exist in engine
[Method: GET, Route: edms/auth]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `auth`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
CRITICAL - 2025-02-26 05:38:41 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'edocs.auth' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `auth`')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `auth`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `auth`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
CRITICAL - 2025-02-26 05:38:41 --> [Caused by] mysqli_sql_exception: Table 'edocs.auth' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `auth`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `auth`')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `auth`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `auth`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
ERROR - 2025-02-26 07:13:58 --> mysqli_sql_exception: Table 'edocs.auth' doesn't exist in engine in C:\xampp\htdocs\CI_CRUD_MVC\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CI_CRUD_MVC\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CI_CRUD_MVC\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CI_CRUD_MVC\app\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CI_CRUD_MVC\app\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
#8 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(933): App\Controllers\UsersController->users()
#9 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#10 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI_CRUD_MVC\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI_CRUD_MVC\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\CI_CRUD_MVC\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\CI_CRUD_MVC\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-02-26 07:13:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'edocs.auth' doesn't exist in engine
[Method: GET, Route: edms/auth]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `auth`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
CRITICAL - 2025-02-26 07:13:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'edocs.auth' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `auth`')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `auth`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `auth`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
CRITICAL - 2025-02-26 07:13:58 --> [Caused by] mysqli_sql_exception: Table 'edocs.auth' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `auth`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `auth`')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `auth`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `auth`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
ERROR - 2025-02-26 08:05:23 --> mysqli_sql_exception: Table 'edocs.auth' doesn't exist in engine in C:\xampp\htdocs\CI_CRUD_MVC\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CI_CRUD_MVC\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CI_CRUD_MVC\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CI_CRUD_MVC\app\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CI_CRUD_MVC\app\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
#8 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(933): App\Controllers\UsersController->users()
#9 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#10 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI_CRUD_MVC\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI_CRUD_MVC\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\CI_CRUD_MVC\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\CI_CRUD_MVC\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-02-26 08:05:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'edocs.auth' doesn't exist in engine
[Method: GET, Route: edms/auth]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `auth`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
CRITICAL - 2025-02-26 08:05:23 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'edocs.auth' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `auth`')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `auth`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `auth`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
CRITICAL - 2025-02-26 08:05:23 --> [Caused by] mysqli_sql_exception: Table 'edocs.auth' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `auth`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `auth`')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `auth`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `auth`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
ERROR - 2025-02-26 08:07:02 --> mysqli_sql_exception: Table 'edocs.auth' doesn't exist in engine in C:\xampp\htdocs\CI_CRUD_MVC\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CI_CRUD_MVC\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CI_CRUD_MVC\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CI_CRUD_MVC\app\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CI_CRUD_MVC\app\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
#8 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(933): App\Controllers\UsersController->users()
#9 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#10 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI_CRUD_MVC\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI_CRUD_MVC\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\CI_CRUD_MVC\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\CI_CRUD_MVC\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-02-26 08:07:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'edocs.auth' doesn't exist in engine
[Method: GET, Route: edms/auth]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `auth`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
CRITICAL - 2025-02-26 08:07:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'edocs.auth' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `auth`')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `auth`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `auth`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
CRITICAL - 2025-02-26 08:07:02 --> [Caused by] mysqli_sql_exception: Table 'edocs.auth' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `auth`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `auth`')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `auth`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `auth`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\UsersModel.php(17): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\UsersController.php(13): App\Models\UsersModel->getUsers()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
ERROR - 2025-02-26 08:10:27 --> mysqli_sql_exception: Table 'edocs.departments' doesn't exist in engine in C:\xampp\htdocs\CI_CRUD_MVC\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\CI_CRUD_MVC\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CI_CRUD_MVC\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CI_CRUD_MVC\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CI_CRUD_MVC\app\Models\DepartmentsModel.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CI_CRUD_MVC\app\Controllers\EDMSController.php(20): App\Models\DepartmentsModel->getDepartments()
#8 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(933): App\Controllers\EDMSController->departments()
#9 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EDMSController))
#10 C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI_CRUD_MVC\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI_CRUD_MVC\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\CI_CRUD_MVC\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\CI_CRUD_MVC\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-02-26 08:10:27 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'edocs.departments' doesn't exist in engine
[Method: GET, Route: edms/departments]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `departments`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\DepartmentsModel.php(15): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\EDMSController.php(20): App\Models\DepartmentsModel->getDepartments()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EDMSController->departments()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EDMSController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
CRITICAL - 2025-02-26 08:10:27 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'edocs.departments' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `departments`')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `departments`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `departments`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\DepartmentsModel.php(15): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\EDMSController.php(20): App\Models\DepartmentsModel->getDepartments()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EDMSController->departments()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EDMSController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
CRITICAL - 2025-02-26 08:10:27 --> [Caused by] mysqli_sql_exception: Table 'edocs.departments' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `departments`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `departments`')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `departments`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `departments`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\DepartmentsModel.php(15): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\EDMSController.php(20): App\Models\DepartmentsModel->getDepartments()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EDMSController->departments()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EDMSController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
DEBUG - 2025-02-26 08:16:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:17:05 --> CSRF token verified.
DEBUG - 2025-02-26 08:17:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:20:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:20:47 --> CSRF token verified.
DEBUG - 2025-02-26 08:20:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:21:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:22:06 --> CSRF token verified.
DEBUG - 2025-02-26 08:22:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:22:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:22:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:22:45 --> CSRF token verified.
DEBUG - 2025-02-26 08:22:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:22:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:22:54 --> CSRF token verified.
DEBUG - 2025-02-26 08:22:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:23:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:23:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:23:53 --> CSRF token verified.
DEBUG - 2025-02-26 08:23:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:26:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:26:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:26:20 --> CSRF token verified.
DEBUG - 2025-02-26 08:26:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:27:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:27:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:28:04 --> CSRF token verified.
DEBUG - 2025-02-26 08:28:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:28:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:28:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:28:59 --> CSRF token verified.
DEBUG - 2025-02-26 08:28:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:29:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:29:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:29:21 --> CSRF token verified.
DEBUG - 2025-02-26 08:29:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:30:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:30:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:30:38 --> CSRF token verified.
DEBUG - 2025-02-26 08:30:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:33:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:33:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:33:26 --> CSRF token verified.
DEBUG - 2025-02-26 08:33:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:34:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:34:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:34:37 --> CSRF token verified.
DEBUG - 2025-02-26 08:34:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:35:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:35:50 --> CSRF token verified.
DEBUG - 2025-02-26 08:35:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 08:35:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:36:05 --> CSRF token verified.
DEBUG - 2025-02-26 08:36:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-26 08:36:27 --> CSRF token verified.
CRITICAL - 2025-02-26 09:19:53 --> ErrorException: Undefined array key "username"
[Method: GET, Route: edms/auth]
in APPPATH\Views\edms\users.php on line 37.
 1 APPPATH\Views\edms\users.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "username"', 'C:\\xampp\\htdocs\\CI_CRUD_MVC\\app\\Views\\edms\\users.php', 37)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\CI_CRUD_MVC\\app\\Views\\edms\\users.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('/edms/users', [], true)
 5 APPPATH\Controllers\UsersController.php(17): view('/edms/users', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
CRITICAL - 2025-02-26 09:19:58 --> ErrorException: Undefined array key "username"
[Method: GET, Route: edms/auth]
in APPPATH\Views\edms\users.php on line 37.
 1 APPPATH\Views\edms\users.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "username"', 'C:\\xampp\\htdocs\\CI_CRUD_MVC\\app\\Views\\edms\\users.php', 37)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\CI_CRUD_MVC\\app\\Views\\edms\\users.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('/edms/users', [], true)
 5 APPPATH\Controllers\UsersController.php(17): view('/edms/users', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
CRITICAL - 2025-02-26 09:21:25 --> ErrorException: Undefined array key "username"
[Method: GET, Route: edms/auth]
in APPPATH\Views\edms\users.php on line 37.
 1 APPPATH\Views\edms\users.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "username"', 'C:\\xampp\\htdocs\\CI_CRUD_MVC\\app\\Views\\edms\\users.php', 37)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\CI_CRUD_MVC\\app\\Views\\edms\\users.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('/edms/users', [], true)
 5 APPPATH\Controllers\UsersController.php(17): view('/edms/users', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
INFO - 2025-02-26 09:25:42 --> CSRF token verified.
INFO - 2025-02-26 09:27:58 --> CSRF token verified.
CRITICAL - 2025-02-26 09:27:58 --> ErrorException: Array to string conversion
[Method: POST, Route: edms/create_user]
in APPPATH\Controllers\UsersController.php on line 35.
 1 APPPATH\Controllers\UsersController.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'C:\\xampp\\htdocs\\CI_CRUD_MVC\\app\\Controllers\\UsersController.php', 35)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->createUser()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
INFO - 2025-02-26 09:29:58 --> CSRF token verified.
CRITICAL - 2025-02-26 09:29:59 --> ErrorException: Array to string conversion
[Method: POST, Route: edms/create_user]
in APPPATH\Controllers\UsersController.php on line 35.
 1 APPPATH\Controllers\UsersController.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'C:\\xampp\\htdocs\\CI_CRUD_MVC\\app\\Controllers\\UsersController.php', 35)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->createUser()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
INFO - 2025-02-26 09:30:31 --> CSRF token verified.
INFO - 2025-02-26 09:31:01 --> CSRF token verified.
INFO - 2025-02-26 09:39:15 --> CSRF token verified.
INFO - 2025-02-26 09:39:39 --> CSRF token verified.
INFO - 2025-02-26 09:47:00 --> CSRF token verified.
INFO - 2025-02-26 10:25:35 --> CSRF token verified.
CRITICAL - 2025-02-26 10:25:36 --> CodeIgniter\Validation\Exceptions\ValidationException: "require" is not a valid rule.
[Method: POST, Route: edms/create_user]
in SYSTEMPATH\Validation\Validation.php on line 346.
 1 SYSTEMPATH\Validation\Validation.php(346): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('require')
 2 SYSTEMPATH\Validation\Validation.php(202): CodeIgniter\Validation\Validation->processRules('password', 'password', null, [...], [...], 'password')
 3 SYSTEMPATH\Controller.php(138): CodeIgniter\Validation\Validation->run()
 4 APPPATH\Controllers\UsersController.php(44): CodeIgniter\Controller->validate([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->createUser()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
INFO - 2025-02-26 10:26:00 --> CSRF token verified.
INFO - 2025-02-26 10:32:46 --> CSRF token verified.
INFO - 2025-02-26 10:53:34 --> CSRF token verified.
INFO - 2025-02-26 11:02:54 --> CSRF token verified.
INFO - 2025-02-26 11:08:11 --> CSRF token verified.
INFO - 2025-02-26 11:08:45 --> CSRF token verified.
INFO - 2025-02-26 11:12:42 --> CSRF token verified.
INFO - 2025-02-26 11:13:52 --> CSRF token verified.
CRITICAL - 2025-02-26 11:13:52 --> ErrorException: Undefined array key "first_name"
[Method: POST, Route: edms/create_user]
in APPPATH\Controllers\UsersController.php on line 59.
 1 APPPATH\Controllers\UsersController.php(59): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "first_name"', 'C:\\xampp\\htdocs\\CI_CRUD_MVC\\app\\Controllers\\UsersController.php', 59)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->createUser()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
INFO - 2025-02-26 11:23:38 --> CSRF token verified.
CRITICAL - 2025-02-26 11:23:38 --> ErrorException: Undefined array key "first_name"
[Method: POST, Route: edms/create_user]
in APPPATH\Controllers\UsersController.php on line 59.
 1 APPPATH\Controllers\UsersController.php(59): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "first_name"', 'C:\\xampp\\htdocs\\CI_CRUD_MVC\\app\\Controllers\\UsersController.php', 59)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->createUser()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
INFO - 2025-02-26 11:24:01 --> CSRF token verified.
CRITICAL - 2025-02-26 11:24:01 --> ErrorException: Undefined array key "role"
[Method: POST, Route: edms/create_user]
in APPPATH\Controllers\UsersController.php on line 59.
 1 APPPATH\Controllers\UsersController.php(59): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "role"', 'C:\\xampp\\htdocs\\CI_CRUD_MVC\\app\\Controllers\\UsersController.php', 59)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->createUser()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
INFO - 2025-02-26 11:27:03 --> CSRF token verified.
CRITICAL - 2025-02-26 11:27:03 --> ErrorException: Undefined array key "role"
[Method: POST, Route: edms/create_user]
in APPPATH\Controllers\UsersController.php on line 59.
 1 APPPATH\Controllers\UsersController.php(59): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "role"', 'C:\\xampp\\htdocs\\CI_CRUD_MVC\\app\\Controllers\\UsersController.php', 59)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->createUser()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
INFO - 2025-02-26 11:33:52 --> CSRF token verified.
CRITICAL - 2025-02-26 11:33:52 --> Error: Call to a member function getValidated() on null
[Method: POST, Route: edms/create_user]
in APPPATH\Controllers\UsersController.php on line 57.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->createUser()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
INFO - 2025-02-26 11:34:56 --> CSRF token verified.
INFO - 2025-02-26 11:37:30 --> CSRF token verified.
INFO - 2025-02-26 11:40:01 --> CSRF token verified.
INFO - 2025-02-26 11:40:38 --> CSRF token verified.
INFO - 2025-02-26 11:44:51 --> CSRF token verified.
CRITICAL - 2025-02-26 11:44:51 --> ErrorException: Undefined array key "password"
[Method: POST, Route: edms/create_user]
in APPPATH\Controllers\UsersController.php on line 77.
 1 APPPATH\Controllers\UsersController.php(77): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "password"', 'C:\\xampp\\htdocs\\CI_CRUD_MVC\\app\\Controllers\\UsersController.php', 77)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->createUser()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
INFO - 2025-02-26 11:45:49 --> CSRF token verified.
INFO - 2025-02-26 13:32:58 --> CSRF token verified.
INFO - 2025-02-26 13:42:26 --> CSRF token verified.
INFO - 2025-02-26 13:44:53 --> CSRF token verified.
INFO - 2025-02-26 13:48:17 --> CSRF token verified.
INFO - 2025-02-26 18:48:53 --> CSRF token verified.
INFO - 2025-02-26 18:49:11 --> CSRF token verified.
INFO - 2025-02-26 18:49:36 --> CSRF token verified.
INFO - 2025-02-26 18:54:03 --> CSRF token verified.
INFO - 2025-02-26 18:58:29 --> CSRF token verified.
CRITICAL - 2025-02-26 18:58:29 --> ArgumentCountError: Too few arguments to function App\Controllers\UsersController::updateUser(), 0 passed in C:\xampp\htdocs\CI_CRUD_MVC\system\CodeIgniter.php on line 933 and exactly 1 expected
[Method: POST, Route: edms/update_user]
in APPPATH\Controllers\UsersController.php on line 89.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->updateUser()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
CRITICAL - 2025-02-26 19:02:26 --> CodeIgniter\HTTP\Exceptions\BadRequestException: The URI you submitted has disallowed characters: "14<"
[Method: POST, Route: edms/update_user/14%3C/form%3E]
in SYSTEMPATH\Router\Router.php on line 739.
 1 SYSTEMPATH\Router\Router.php(207): CodeIgniter\Router\Router->checkDisallowedChars('edms/update_user/14</form>')
 2 SYSTEMPATH\CodeIgniter.php(832): CodeIgniter\Router\Router->handle('edms/update_user/14</form>')
 3 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
INFO - 2025-02-26 19:04:02 --> CSRF token verified.
INFO - 2025-02-26 19:04:50 --> CSRF token verified.
DEBUG - 2025-02-26 19:09:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-02-26 19:09:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-02-26 19:09:26 --> Error: Call to undefined function validation_list_errors()
[Method: GET, Route: edms/auth]
in APPPATH\Views\edms\users.php on line 6.
 1 SYSTEMPATH\View\View.php(224): include()
 2 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('/edms/users', [], true)
 4 APPPATH\Controllers\UsersController.php(17): view('/edms/users', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
DEBUG - 2025-02-26 19:09:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-02-26 19:09:43 --> Error: Call to undefined function validation_list_errors()
[Method: GET, Route: edms/auth]
in APPPATH\Views\edms\users.php on line 6.
 1 SYSTEMPATH\View\View.php(224): include()
 2 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('/edms/users', [], true)
 4 APPPATH\Controllers\UsersController.php(17): view('/edms/users', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsersController->users()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CI_CRUD_MVC\\public\\index.php')
